webapp/packages/core-ui/src/Tabs/TabsState.tsx

```diff
@@ -186,6 +189,12 @@ export const TabsState = observer(function TabsState<T = Record<string, any>>({
           props: dynamic.props,
         });
       },
+      async closeTabGroup(tabId: string) {
+        dynamic.onCloseTabGroup?.({
+          tabId,
+          props: dynamic.props,
+        });
+      },
```

Comment:
the current implementation uses common `await this.close(tab);` to close the tab because it will trigger logic with `canClose`
seems like this feature is not a part of tabs because they don't have the concept of "groups"

it will be better to revert all these changes

there are two ways:
1. add tabs group concept to the Tabs (we don't have many cases for this, so probably not)
2. implement this action as an extension for Tabs form sql-editor: I mean that SQL editor will register the menu creator and action handler and will create this new action for closing tabs of the same query. `TabMenu` accepts `menuContext` that can be used to add extra context to tabs and use it in menu creators and action handlers later